### DISCUSSION TITLE
G0W0 Problem

### DISCUSSION THREAD

**Discussion by LAZAARKOUSSAI** [sblisesivdin/gpaw-tools#77] [Q&A] [answered]
MoS2 example error:  
*********************
Number of atoms imported from CIF file:3
Spacegroup of CIF file (from SPGlib): Amm2 (38)
Special Points usable for this spacegroup: {'G': array([0., 0., 0.]), 'A': array([0. , 0. , 0.5]), 'H': array([0.33333333, 0.33333333, 0.5       ]), 'K': array([0.33333333, 0.33333333, 0.        ]), 'L': array([0.5, 0. , 0.5]), 'M': array([0.5, 0. , 0. ])}
Passing ground state calculation for GW calculation...
Traceback (most recent call last):
  File "/home/....../gpaw-tools-main/gpawsolve.py", line 1842, in <module>
    gpawsolver.groundcalc()
  File "/home/....../gpaw-tools-main/gpawsolve.py", line 383, in groundcalc
    gw = G0W0(struct+'-1-Result-Ground.gpw', filename=struct+'-1-', bands=(GW_valence_band_no, GW_conduction_band_no),
  File "/home/....../.local/lib/python3.10/site-packages/gpaw/response/g0w0.py", line 1125, in __init__
    super().__init__(filename=filename,
TypeError: G0W0Calculator.__init__() got an unexpected keyword argument 'method'

**Comment by sblisesivdin** [comment, marked as answer]
GW0 feature has been removed from GPAW as pointed out in https://wiki.fysik.dtu.dk/gpaw/releasenotes.html#version-22-8-0. 

Because it is not mature enough with its current case, it may be better to remove GW from gpaw-tools.
